circle.yml

```diff
@@ -42,6 +42,14 @@ jobs:
           root: .
           paths: bin
 
+  run:
+    <<: *defaults
+    steps:
+      - checkout
+      - attach_workspace:
+          at: /go/src/github.com/gruntwork-io/aws-nuke
+      - run: go run main.go
```

Comment:
Shouldn't this exclude `us-west-2`? And only delete resources older than, say, 2 days?

Reply:
I think 2 days is fair, although I'm interested to know how you arrived at that figure because I was gonna go with 24hrs. What's the frequency at which resources are typically created in the Phx DvOps account?

Reply:
2 days is just a guess. We may have to tweak it later. Resources that are left from tests would ideally be cleaned up nightly. However, we also use Phx DevOps for manual testing, and occasionally, we leave things running a bit longer.